templates/accessibility.html

```diff
@@ -5,71 +5,82 @@
 
 {% block content %}
-<div id="main-content" class="p-strip--image is-dark vfio-hero">
-  <div class="row">
-    <h1 class="u-no-margin--bottom">Accessibility guidelines</h1>
+<div class="p-strip--light is-deep">
+  <div class="u-fixed-width">
+    <h1 class="p-heading--1">Accessibility guidelines</h1>
+    <p class="p-heading--">Vanilla Framework aims for Level AA conformance with the <br><a href="https://www.w3.org/TR/WCAG21/" class="p-link--external">Web Content Accessibility Guidelines (WCAG) 2.1</a></p>
   </div>
 </div>
-<div class="p-strip is-bordered">
+<div class="p-strip">
   <div class="row">
-    <div class="col-8">
-      <h2 class="p-heading--3">Vanilla aims to be as inclusive as possible.</h2>
-      <p>While traditionally accessibility focuses on making the web more accessible for users with permanent disabilities, a focus on accessibility can deliver an improved user experience for everyone, including those with a temporary or circumstantial disability.</p>
-      <p>Accessibility should permeate the entire design and development process, rather than being something that you only think about after everything else has been defined and implemented. You should focus on accessibility when you are:</p>
-      <ul>
-        <li>Writing your content (whether that’s words, images, videos or any other media format)</li>
-        <li>Designing and building your components</li>
-      </ul>
-      <p>While the accessibility of some of the existing Vanilla design patterns is still being improved, all new and updated patterns should meet <a href="https://www.w3.org/TR/WCAG20/" class="p-link--external">WCAG 2.0</a>level AA.</p>
+    <div class="col-6">
+      <h3 class="p-heading--2">Ensuring conformance</h3>
+      <p>We use the following tools to continually audit the framework:</p>
+    </div>
+    <div class="col-6">
+      <p class="p-heading--5 u-no-margin--bottom"><a href="https://www.w3.org/WAI/eval/report-tool/#!/evaluation/audit#%2F" class="p-link--external">Accessibility report tool</a></p>
+      <p>A checklist that can be filtered by A / AA / AAA level, with a short description and links to the related "Understanding" and "How to Meet" articles that accompany each criterion.</p>
+      <p class="p-heading--5 u-no-margin--bottom"><a href="https://www.w3.org/TR/wai-aria-practices" class="p-link--external">WAI-ARIA Authoring Practices 1.1</a></p>
+      <p>
+        <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>A guide for understanding how to use <cite><a href="https://www.w3.org/TR/wai-aria-1.1/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> 1.1</a></cite> to create an accessible Rich Internet Application. It provides guidance on the appropriate application of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>, describes recommended <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> usage patterns, and explains concepts behind them.</p>
```

Comment:
Isn't there one A too much in first acronym? 

<img width="648" alt="Screenshot 2020-10-09 at 09 26 23" src="https://user-images.githubusercontent.com/83575/95555112-95454a00-0a11-11eb-88f2-71996842ad55.png">